.github/workflows/main.yml

```diff
@@ -65,12 +65,26 @@ jobs:
       # Compilation and registration of the PyCOMTest server dll
       - name: Set up MSVC
         uses: microsoft/setup-msbuild@v2
+
+      # See https://github.com/actions/runner-images/issues/9701
+      # Adapted from https://github.com/actions/runner-images/issues/9873#issuecomment-2139288682
+      - name: Install missing Visual Studio components
+        run: |
+          Set-Location "C:\Program Files (x86)\Microsoft Visual Studio\Installer\"
+          $VsInstallPath = vswhere.exe -latest -products * -requires Microsoft.Component.MSBuild -property installationPath
+          [string]$ComponentsToAdd = @(
+            "Microsoft.VisualStudio.Component.VC.14.29.16.11.ATL"
```

Comment:
I tried to get info about the CI's VS and MSVC versions and this is what I got using `vswhere -latest`
https://github.com/mhammond/pywin32/actions/runs/14687605417/job/41218346703?pr=2582#step:6:6
```
VS_PATH: C:\Program Files\Microsoft Visual Studio\2022\Enterprise
MSVC_VERSION: 14.43
VS_VERSION: 17.13
```

Which is completely different than the `14.29` and `16.11` needed here. That's why I hardcoded this component version number.